charts/spire/charts/spire-server/values.yaml

```diff
@@ -344,7 +344,7 @@ nodeAttestor:
     enabled: true
     serviceAccountAllowList: []
 
-# tornjak - Tornjak specific configuration
+# tornjak - Tornjak default values
```

Comment:
All values in this file are the defaults, I think it is not worthwhile to mention the values file are defaults.